samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1051,6 +920,315 @@ or
+   You will use them to sign device credentials.
+
+   If you were directed here as part of other instructions, proceed to the next step of those instructions.
+
+   .. note::
+      You only need to generate these three files once.
+      They can be used to sign as many device credentials as you need.
+
+.. _nrf_cloud_multi_service_create_device_cred_locally:
+
+Generating device credentials locally
+=====================================
+
+To generate and sign a device credential locally using your :ref:`device-owner CA cert <nrf_cloud_multi_service_create_device_owner_ca>`, perform the following steps:
+
+1. Create a globally unique `device ID <nRF Cloud Device ID_>`_ for the device.
+
+   The ID can be anything you want, as long as it is not prefixed with ``nrf-`` and is globally unique.
+   Alternatively, if your device is an LTE development kit sold by Nordic, you can use ``nrf-<IMEI>`` where ``<IMEI>`` is the IMEI of the device.
+   See `nRF Cloud Device ID`_ for details.
+
+   Each device should have its own device ID, and you must use this device ID exactly for all other actions involving that device, otherwise onboarding will fail.
+   We will refer to this ID in other steps using the term ``<device_uuid>``.
+
+#. Navigate to the folder where you :ref:`created your device-owner CA cert <nrf_cloud_multi_service_create_device_owner_ca>`, and use the :file:`create_device_credentials.py` Python script :ref:`you installed <nrf_cloud_multi_service_install_nrf_utils>` to generate a self-signed certificate for the device:
+
+   .. code-block:: console
+
+      python3 create_device_credentials.py -ca "owner_<own_cert_serial>_ca.pem" -ca_key "owner_<own_cert_serial>_prv.pem" -c US -cn "<device_uuid>" -f cred_
+
+   Where ``<device_uuid>`` is the device ID you created, and :file:`owner_<own_cert_serial>_ca.pem`, and :file:`owner_<own_cert_serial>_prv.pem` are the device-owner CA certificate and private key files.
+
+   Remember to set ``-c`` to your two-letter country code.
+   See the `Create Device Credentials <nRF Cloud Utilities Create Device Credentials_>`_ section in the nRF Cloud Utilities documentation for more details.
+
+   You should now have the following three files:
+
+     * :file:`cred_<device_uuid>_crt.pem`
+     * :file:`cred_<device_uuid>_pub.pem`
+     * :file:`cred_<device_uuid>_prv.pem`
+
+   These three files are your device credentials, and can only be used by a single device.
+
+   If you were directed here as part of other instructions, proceed to the next step of those instructions.
+
+   .. important::
+
+      If an attacker obtains the private key contained in :file:`cred_<device_uuid>_prv.pem`, they will be able to impersonate your device.
+
+.. _nrf_cloud_multi_service_onboard_device_manually:
+
+Onboarding a device manually
+============================
+
+Once you have obtained device credentials for your device, it can be `onboarded <nRF Cloud Onboarding_>`_.
```

Comment:
```suggestion
Once you have obtained the device credentials for your device, it can be `onboarded <nRF Cloud Onboarding_>`_.
```

Reply:
Because these credentials do not exist before they are obtained, using `the` here feels very unnatural